inst/pages/97_extra_materials.qmd

```diff
@@ -639,7 +639,7 @@ under altExp.
 
 Tax_glom removes the taxa which have not been assigned to the level given in
 taxrank by default (NArm = TRUE).
-So we will add the onRankOnly = TRUE to mergeFeaturesByRank function which is
+So we will add the disable.taxonomy = TRUE to mergeFeaturesByRank function which is
 equivalent to the default behaviour of tax_glom.
```

Comment:
No this is not correct. disable.taxonomy should not be used. na.rm =TRUE is the option to use (which is already the default value)